sixtyfps_runtime/interpreter/eval.rs

```diff
@@ -1079,3 +1097,16 @@ pub fn default_value_for_type(ty: &Type) -> Value {
         }
     }
 }
+
+lazy_static::lazy_static! {
+    static ref INTERNED_STRINGS: std::sync::Mutex<std::collections::HashSet<String>> = Default::default();
+}
+
+pub fn interned_str(s: &str) -> &'static str {
+    let mut cache = INTERNED_STRINGS.lock().unwrap();
+    if !cache.contains(s) {
+        cache.insert(s.into());
+    }
+    let s: &str = cache.get(s).unwrap();
+    return unsafe { std::mem::transmute(s) };
```

Comment:
I don't think that's valid.
The String could be moved when the Hash is resized and the String might implement small string optimization in the future, who knows.